pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -163,6 +163,15 @@ func (c *Cluster) checkHealth(ctx context.Context) error {
 		return errors.New("skipping mon health check since there are no monitors")
 	}
 
+	monsToSkipReconcile, err := c.getMonsToSkipReconcile()
```

Comment:
should we add a comment like 

\\ skip reconcile if "ceph.rook.io/do-not-reconcile": "true" is added to mon pod

Reply:
Yeah I will mention the exact label in the logging like on line 171 so it is clear, I was thinking about that as well in my last test.